init.sh

```diff
@@ -73,7 +73,7 @@ echo "Setting up Major ... "
 MAJOR_VERSION="1.3.4"
 MAJOR_URL="http://mutation-testing.org/downloads"
```

Comment:
Let's change this also to https.

Reply:
Also, let's change all UMass/UW-specific urls to defects4j.org (e.g., line 22) -- the files and certificates are already in place: `https://defects4j.org/downloads` (Note that the new url uses downlads, as opposed to download)